modules/guac/ra_guac_autoscale_public_alb.template.cfn.yaml

```
@@ -656,6 +656,12 @@ Resources:
               cfn-signal -e 1 --stack ${AWS::StackName} --resource AutoScalingGroup --region ${AWS::Region} ; \
               exit 1 ; \
               )
+      LaunchTemplateName: !Ref AWS::StackName
+      TagSpecifications:
+        - ResourceType: "launch-template"
```

Let's remove the quotes. They aren't necessary in yaml for simple strings like this...
```suggestion
        - ResourceType: launch-template
```